### PR TITLE
ISP filtering issues when cloning

### PR DESCRIPTION
**Pull request by grahamehorner**
A friendly note/point that may help when ISPs having web filtering that drops/kills connections

Summary of the changes (Less than 80 chars)
 - Detail 1
 - Detail 2

Addresses #bugnumber (in this specific format)

### CODE REVIEW

File: docs/BuildFromSource.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ To update an existing copy, run:
 git submodule update --init --recursive
 ```
 
+**NOTE** some ISPs have been know to use web filtering software that has caused issues with git repository cloning, if you experiance issues cloning this repo please review <https://help.github.com/en/github/authenticating-to-github/using-ssh-over-the-https-port>
```
Comment (dougbu) [inline docs/BuildFromSource.md:71]:
```suggestion
**NOTE** some ISPs have been known to use web filtering software that has caused issues with git repository cloning, if you experiance issues cloning this repo please review <https://help.github.com/en/github/authenticating-to-github/using-ssh-over-the-https-port>
```

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
Thanks @grahamehorner❕ Did this change fix an issue you filed❔ (I'd like to close it if so.)

**Comment by dougbu** [comment]
For example, is his a complete fix for #19905❔

**Comment by grahamehorner** [comment]
@dougbu yes the above issue was resolved when following the guidelines pointed to in the url.  The ISP confirmed the their web filtering was terminating https/ssh connection and when using ssh over https all worked as expected.  This took a little investigation as not all git repos had the same issue or triggered the web filtering to terminate.

**Pull request merged**